lapis/src/main/kotlin/org/genspectrum/lapis/openApi/OpenApiDocs.kt

```diff
@@ -196,6 +211,17 @@ fun buildOpenApiSchema(
                             ),
                     ),
                 )
+                .addSchemas(
+                    MOST_RECENT_COMMON_ANCESTOR_RESPONSE_SCHEMA,
+                    lapisArrayResponseSchema(
+                        Schema<String>()
+                            .types(setOf("object"))
+                            .description(
+                                "The response contains the most recent common ancestor node label `mrcaNode`, the number of nodes in the filter not in the phylotree `missingNodeCount` and optionally their names `missingFromTree`.",
+                            )
```

Comment:
Do we need this description? AFAICS all of the information here is also already in the description of the fields of this object.